BlazarUI/app/scripts/components/repo-build/RepoBuildContainer.jsx

```diff
@@ -31,7 +35,9 @@ let initialState = {
   loadingModuleBuilds: true,
   loadingRepoBuild: true,
   loadingStars: true,
-  branchInfo: {}
+  branchInfo: {},
+  upAndDownstreamModules: {},
+  currentRepoBuild: undefined
```

Comment:
Can we specifically use `null` when something is purposely not set? `undefined` to me means that you have a object and because of some coding error, it's missing a property that should exist